### ISSUE TITLE
feature: allow to disable report bug button per project/package

### ISSUE THREAD

**Issue by plusky** [openSUSE/open-build-service#4241]
While there are indeed some usecases for having such button, it also misused. There are bugs reported for packages that are not yet production ready/not intended for general usage. Having bugreports for such packages is rarely helpful while presence of said button creates false impression of support of such packages/projects. I would like to ask for having possibility to disable "report bug" per package/project to avoid such misunderstandings.

**Comment by ChrisBr** [comment]
@adrianschroeter currently the ``report bug`` button is shown when the bugowner email is set.

**Comment by adrianschroeter** [comment]
Right, so why not removing the bugowner if you don't want to have one? @plusky

**Comment by ChrisBr** [comment]
I mean one disadvantage is that you would need to remove it from the project which means there is not such a button for any package inside this project...

**Comment by adrianschroeter** [comment]
right, but you are either responsible for all or not. We don't allow to disable other roles as well per package, so why should we do so only for bugowner role?

This sounds not understandable for the user and is IMHO a project setup problem in first place.

**Comment by plusky** [comment]
@adrianschroeter well I want to have bugowner set for bugzilla triaging i.e for factory/Leap, but I do not want to create impression that something found in devel project is supposed to be working.
